src/Akeneo/Pim/Enrichment/Bundle/Controller/InternalApi/SequentialEditController.php

```diff
@@ -62,8 +62,10 @@ public function getIdsAction(Request $request): JsonResponse
 
         $cursor = $this->getProductsCursor($filters, [
             'locale' => $parameters['dataLocale'],
-            'scope'  => $parameters['dataScope']['value'],
-            'sort'   => $parameters['sort']
+            'scope' => isset($parameters['dataScope'])
+                ? $parameters['dataScope']['value']
+                : null,
```

Comment:
```suggestion
            'scope' => $parameters['dataScope']['value'] ?? $parameters['dataScope'] ?? null,
```
To handle all the possible cases

Reply:
Revisited application logic after discussion with Romain, since i was not sure on the solution....